### DISCUSSION TITLE
Página con información de cómo conseguir un certificado?

### DISCUSSION THREAD

**Discussion by palmerabollo** [PucelaBits/websegura#131] [Ideas]
Podríamos tener una página con info de cómo conseguir un certificado. Por ejemplo, enlazando a Letsencrypt. Tengo dudas de si es interesante, porque normalmente los problemas no son técnicos sino de falta de voluntad.

**Comment by nukeador** [comment]
Yo creo que a equipo técnico que le mandan poner HTTPs de un ministerio sabe perfectamente cómo hacerlo. Lo único que en nuestra web en el informe técnico le recuerda los estándares actuales de seguridad.
